install.sh

```diff
@@ -1,79 +1,69 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
-
-echo "Runnning Sniprun Installer"
-local_version=v$(cat Cargo.toml | grep ^version | cut -d "\"" -f 2)
+arch=$(uname)
 name="sniprun"
-force_build=$1
+local_version="v$(grep "^version" Cargo.toml | cut -d '"' -f 2)"
+
+if [ "$1" = "-f" ] || [ "$1" = "--force" ]; then
```

Comment:
while conceptually better than just checking if there is an argument to the script, this would break many configs, and this is not even proper parsing (it would have to be modified to accomodate other arguments). 

I don't think this is needed

Reply:
> and this is not even proper parsing (it would have to be modified to accomodate other arguments)

I know but does the script more elaborate parsing? You were checking the first argument of the script, that could be anything. This is very unpredictable and it is a bad design choice handling this type of input. 

I've chosen `-f` because it is a common pattern in cli application, but we could make it so if the user input anything other than one the script will exit and print an usage string.

```bash
# if $1 is not empty and not 1 exit
if [ -n $1 ] && [ $1 != 1 ]; then
  echo "Usage: install.sh <1>"
  echo "1: force install"
  exit 1
fi
```